src/vhip/mlmodel/compute_ml_features.py

```diff
@@ -459,6 +476,35 @@ def compute_feature(self, pair: Pairs) -> Pairs:
 
         pair.GCdifference = self.GCcontent[pair.virus] - self.GCcontent[pair.host]  # pyright: ignore
 
+        # Create CodonBiasComparison objects for the pair
+        virus = pair.virus.replace(
+            self.genome_ext, self.gene_ext
+        )  # first get the correct key based on genes file name (virus)
+        host = pair.host.replace(
+            self.genome_ext, self.gene_ext
+        )  # first get the correct key based on genes file name (host)
+        pair.codons_comparison = CodonBiasComparison(
+            self.codon_frqs[virus],  # pyright: ignore
```

Comment:
what are the pyright: ignore for?

Reply:
Thanks for asking! I realized there was a better way to solve the issue - I added typing to some dictionaries and removed those statements. There are a bunch of pre-existing ignore statements right above this - I started trying to get rid of the need for these, but maybe you remember your justification for them and can save me some time if so?